### PR TITLE
PR: search by prefecture map implementation

### PR DESCRIPTION
**Pull request by harunakawakami**
## Summary
- Add a map plugin to search by prefecture page

## Checklist
- [x] I fixed bugs that the map did not show when loading from the landing page to the map page and browser back to the map page.
- [x] No error nor warning in the console.
- [x] Successfully sent the data to the search result page

Search by Map page
image

Search Result page
image
image

Issue - #9

### CODE REVIEW

File: global.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+declare function svgJapan(options: any): any;
```
Comment (Tricole) [inline global.d.ts:1]: - Looks good, but if possible please try to include a different type to any

**Review by Tricole — CHANGES_REQUESTED** [review]
Overall, it looks really clean and is clear. Just one small point for the TS types!

File: global.d.ts (hunk 1 above)
Comment (harunakawakami) [reply global.d.ts]: Hi Kamil. Thank you for your advice. I fixed the global.d.ts file and specified the type from any.

**Review by Tricole — APPROVED** [review]
LGTM!

**Pull request merged**